.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+# Run all tests for commits to master
+
+name: bash / master
+
+on:
+  push:
+    branches:
+      - master
+      - main
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+      - name: Install bats-core
+        run: npm install -g bats
+
+      - name: Run tests for all exercises
+        run: bash .github/scripts/ci
```

If you're calling a script, it's a good idea to run with the following options:
- xtrace (outputs each command that is run, useful for debugging)
- errexit (aborts script with exit code 1 if any command has a non-zero exit code)
	- Does not exit if the failing command is the condition for an `if` or `while`
```suggestion
        run: bash -xe .github/scripts/ci
```

---

This `ci` script invokes other scripts that already use `-e`, so not necessary to put that option here. 

I get where you're coming from regarding`-x`, but I don't think it would add much value.